util/buildRelease/testRelease

```diff
@@ -85,12 +90,20 @@ if ($debug == 1) {
 
 
 #
-# set mail options
+# set mail options. Default to util/test/send_email.py, if available and
+# working. If not available or not working, default to 'mail'.
 #
 $mailer = $ENV{'CHPL_MAILER'};
 if ($mailer eq "") {
-    $mailer = "mail";
+    $chplsendemail = "$chplhomedir/util/test/send_email.py";
+    `$chplsendemail --help >/dev/null 2>&1`;
+    if ($? == 0) {
+        $mailer = "$chplsendemail --header=Reply-To=$replymail,Precedence=bulk";
+    } else {
+        $mailer = "mail";
+    }
 }
+print "\$mailer = $mailer\n";
```

Comment:
Maybe something like ' print "[Error send_email.py not found or not working correctly]\n" ' in the else where mailer is set to mail